daffodil-runtime1/src/main/scala/edu/illinois/ncsa/daffodil/processors/parsers/PrimitivesTextNumber1.scala

```diff
@@ -55,33 +55,33 @@ import edu.illinois.ncsa.daffodil.processors.Dynamic
 import edu.illinois.ncsa.daffodil.processors.ElementRuntimeData
 import edu.illinois.ncsa.daffodil.processors.Evaluatable
 import edu.illinois.ncsa.daffodil.processors.ParseOrUnparseState
-import edu.illinois.ncsa.daffodil.processors.RuntimeData
 import edu.illinois.ncsa.daffodil.processors.Success
 import java.lang.{ Number => JNumber }
 import java.math.{ BigDecimal => JBigDecimal }
 import java.math.{ BigInteger => JBigInt }
+import edu.illinois.ncsa.daffodil.processors.TermRuntimeData
 
 case class ConvertTextCombinatorParser(
-  rd: RuntimeData,
+  rd: TermRuntimeData,
   valueParser: Parser,
   converterParser: Parser)
   extends ParserObject(rd) {
 
   override lazy val childProcessors = Seq(valueParser, converterParser)
 
   def parse(start: PState): Unit = {
-    valueParser.parse1(start)
+    start.parse1(valueParser)
```

Comment:
fixup here also. When parse1 disappears from PState all these will be identified by the scala compiler.